src/test/java/org/commcare/formplayer/repo/FormSessionRepoTest.java

```diff
@@ -198,6 +202,25 @@ public void testDeleteByDateCreatedLessThan() {
         assertThat(remainingIds.size()).isEqualTo(3);
     }
 
+    @Test
+    public void testFormDefinitionRelationship() {
+        SerializableFormDefinition formDef = new SerializableFormDefinition(
+                "appId",
+                "formXmlns",
+                "formVersion",
+                "formXml"
+        );
+        this.formDefinitionRepo.save(formDef);
+        SerializableFormSession session = getSession();
+        session.setFormDefinition(formDef);
+        this.formSessionRepo.save(session);
+        SerializableFormDefinition fetchedFormDef = this.formDefinitionRepo.getById(formDef.getId());
```

Comment:
shouldn't this test be checking that you can get the SerializableFormDefinition from the SerializableFormSession without having to fetch it directly?